docs/printing-checklist.md

```diff
@@ -0,0 +1,18 @@
+# Printing
+
+<!-- SPDX-License-Identifier: CC0-1.0 -->
+<!-- SPDX-FileCopyrightText: 2021-2024 The Foundation for Public Code <info@publiccode.net>, https://standard.publiccode.net/AUTHORS -->
+
+The printed Standard for Public Code checklist is printed by Reclameland.
+This guide tries to provide the relevant information to print it there or somewhere else.
+
+Details, mostly in order they are on the Reclameland page with the Dutch if necessary:
```

Comment:
```suggestion
Details, mostly in the order they are on the Reclameland page with the Dutch if necessary:
```

Reply:
As this is the printing checklist, it's not part of the PDF, but we should fix it.